src/pages/BorrowerProfile/BorrowerProfile.vue

```diff
@@ -307,7 +308,13 @@ export default {
 					vmid: 'twitter:description',
 					content: this.shareDescription
 				},
-			])
+			]).concat(isSclePresent ? [
+				{
+					vmid: 'canonical',
+					rel: 'canonical',
+					href: `https://${this.$appConfig.host}${this.$route.path}?${this.$route.query?.utm_campaign}`
```

Comment:
I think we need to include the entirety of the query param set here? @emuvente thoughts?

Reply:
It's best practice in canonical urls to only include parameters that change the content of the page, because a different page could be indexed for every different set of parameters listed in the canonical. If a bunch of pages are indexed that have the same content, the overall ranking of the base page will be lowered, because it's not unique content anymore.

That said, we could also use `noindex` to prevent indexing and then use the entire query string in the canonical url, which would future-proof this from any parameter naming changes and avoid anything besides the base url being indexed. I think that's probably the best way to go:
```js
]).concat(isSclePresent ? [
	{
		vmid: 'robots',
		name: 'robots',
		content: 'noindex',
	},
] : []),
link: (isSclePresent ? [
	{
		vmid: 'canonical',
		rel: 'canonical',
		href: `https://${this.$appConfig.host}${this.$route.fullPath}`,
	},
] : []),
```

Reply:
I think the approach you've outlined is the best way for now and I look forward to cleaning this out of the code.

Reply:
@edwinf-barreram Please address the note above from @emuvente

Reply:
Done! @mcstover